vignettes/experiments-with-R/slide-code.R

```diff
@@ -0,0 +1,193 @@
+#### Code from slides
```

Comment:
Are these for a presentation slide deck?  If so we probably should not include in vignettes

Reply:
Removed file from PR -- it should not have been included.